### DISCUSSION TITLE
How to define region in s3 config file for seaweedfs s3?

### DISCUSSION THREAD

**Discussion by MadalasaP** [seaweedfs/seaweedfs#4345] [Q&A]
Currently, seaweed-fs s3 is picking up default region us-east-1. But want to change region to us-west-1.

started seaweedfs s3 by the following command
./weed server -dir=/bryck/seaweed -master.volumeSizeLimitMB=1024 -master.volumePreallocate=false -s3 -s3.port 8333 -s3.config /home/bucket/.s3_config

s3_config file
`{
    "identities": [
        {
            "name": "me",
            "credentials": [
                {
                "accessKey": "any",
                "secretKey": "anylongkey"
                }
            ],
            "actions": [
                "Read",
                "Write",
                "List",
                "Tagging",
                "Admin"
            ]
        }
    ]
}`

Receiving the following error:
**S3 bucket seaweedfs: Failed to update region for bucket: reading bucket location failed: AccessDenied: Access Denied**

**Comment by chrislusf** [comment]
does not seem a SeaweedFS error message

**Reply by MadalasaP** [reply]
Received warning **NOTICE: S3 bucket seaweedfs: Switched region to "us-west-1" from "us-east-1"**
